project-board-crew-logic/src/main/java/project/board/crew/logic/structure/crew/CrewController.java

```diff
@@ -0,0 +1,75 @@
+package project.board.crew.logic.structure.crew;
+
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import project.board.crew.logic.structure.categories.CrewCategories;
+import project.board.crew.logic.structure.user.Users;
+
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api/crew")
+public class CrewController {
+
+    @Autowired
+    private CrewService crewService;
+
+    @PostMapping()
+    public ResponseEntity<List<Crew>> getCrewsList()
+    {
+       return ResponseEntity
+               .status(HttpStatus.OK)
+               .body(crewService.getCrews());
+    }
+
+    @PostMapping(value = "/read-crew", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Crew> getCrew(@ModelAttribute Crew crew)
```

Comment:
Nie wiem o co chodzi z tym get Crew by crew XD powinieneś mieć metody get na przykład po id albo name. Popraw mnie jeśli to ma sens